plugin/pkg/scheduler/scheduler.go

```diff
@@ -107,6 +107,9 @@ func New(c *Config) *Scheduler {
 
 // Run begins watching and scheduling. It starts a goroutine and returns immediately.
 func (s *Scheduler) Run() {
+	go util.Forever(func() {
+		metrics.BindingRateLimiterSaturation.Set(s.config.BindPodsRateLimiter)
```

Comment:
@lavalamp 
I didn't know well enough all those utility functions. Thanks for suggesting it. It actually looks clearer simpler in this way.